crates/typed-store-macros/src/lib.rs

```diff
@@ -218,6 +293,13 @@ pub fn derive_dbmap_utils(input: TokenStream) -> TokenStream {
                     _ => anyhow::bail!("No such table name: {}", table_name),
                 })
             }
+
+            /// This gives info about memory usage
+            fn get_memory_usage(&self) -> Result<(u64, u64), typed_store::rocks::TypedStoreError> {
```

Comment:
Thanks much for this.  Maybe add `returns (total table memory, cache memory)` to the doc?